### ISSUE TITLE
Large Commit ID is persisted with different value than logged

### ISSUE THREAD

**Issue by andreiamariei** [javers/javers#855]
When using RANDOM CommitIdGenerator, Javers logs different Commit IDs than what's being actually persisted in the database.

The issue seems to be the conversion to `double` in `CommitId.valueAsNumber()`.

Test (quickly written in spock):
```
def "large commit id test"() {
    expect:
        new CommitId(8804939115786087745, 0).valueAsNumber() == 8804939115786087745.00
}
```
results in:
```
Condition not satisfied:

new CommitId(8804939115786087745, 0).valueAsNumber() == 8804939115786087745.00
|                                    |               |
8804939115786087745.0                |               false
                                     8804939115786087424.00

```

**Comment by bartoszwalacik** [comment]
Hi , we will be happy to accept your pull request with the fix

**Comment by bartoszwalacik** [comment]
fixed in 5.6.1
